src/components/UI/Progress/CircularProgress.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="relative">
+    <svg :height="radius * 2" :width="radius * 2">
+      <!-- background circle -->
+      <circle
+        stroke="#C4C5C5"
+        :stroke-dasharray="circumference + ' ' + circumference"
+        :style="{ strokeDashoffset: 0 }"
+        :stroke-width="stroke"
+        fill="transparent"
+        :r="normalizedRadius"
+        :cx="radius"
+        :cy="radius"
+      />
+      <!-- the actual progress circle -->
+      <circle
+        stroke="#2B9D05"
+        :stroke-dasharray="circumference + ' ' + circumference"
+        :style="{ strokeDashoffset: strokeDashoffset }"
+        :stroke-width="stroke"
+        fill="transparent"
+        :r="normalizedRadius"
+        :cx="radius"
+        :cy="radius"
+      />
+    </svg>
+    <!-- progress indicator in the center of the circle  -->
+    <div
+      v-if="toShowNumberIndicator"
+      class="absolute inset-1/3 flex flex-col justify-center"
+      data-test="numberIndicator"
+    >
+      <div class="text-sm sm:text-base md:text-2xl lg:text-4xl font-bold text-center">
+        {{ Math.trunc(progress) }}%
```

Comment:
progress is not the right word no? this is more of a result.

the more appropriate word would simply be accuracy?